src/main/java/org/craftercms/studio/impl/v1/service/deployment/PublishingManagerImpl.java

```diff
@@ -116,6 +117,13 @@ public DeploymentItemTO processItem(PublishRequest item) throws DeploymentExcept
         String user = item.getUser();
 
         String liveEnvironment = LIVE_ENVIRONMENT;
+
+        boolean publishedEnvironmentsEnabled = Boolean.parseBoolean(
```

Comment:
Change to new constant and name. Note that new constant is the reverse of the old (true/false).